frontend/app/components/transportation/study-area-map/study-selection-toggler.js

```diff
@@ -34,13 +34,14 @@ export default class TransportationStudyAreaMapStudySelectionTogglerComponent ex
   async toggleCensusTract(selectedCensusTractFeatureArray) {
     const analysis = await this.analysis;
     const existingStudySelection = analysis.get('jtwStudySelection');
+    const requiredStudySelection = analysis.get('requiredJtwStudySelection');
     // check that selectedCensusTractFeature array exists and has an item
     if (selectedCensusTractFeatureArray && selectedCensusTractFeatureArray.length) {
       let { geoid } = selectedCensusTractFeatureArray[0].properties || {};
-      // check that the feature has a geoid property
-      if(geoid) {
+      // check that the feature has a geoid property and is not part of the required selection
+      if(geoid && !requiredStudySelection.includes(geoid)) {
         if(existingStudySelection.includes(geoid)){
-          existingStudySelection.popObject(geoid);
+          existingStudySelection.removeObject(geoid);
```

Comment:
this was a bug; we were just popping the last object that had been added. Now it actually removes the requested objected. Added test coverage for this.